### PR TITLE
[WIP] feat(website/test): with a command i can generate snapshots for all previews (#199)

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [X] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [ ] Yes
- [X] No

## Describe the changes

Github issue number or Jira issue URL: N/A

https://design-system-adeo.atlassian.net/browse/MZC-199

### CODE REVIEW

File: test/previewsPath.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+module.exports = [
```
Comment (gael-boyenval) [inline test/previewsPath.js]: we need to generate that

File: test/spec/screenshots.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+require('dotenv').config()
```
Comment (gael-boyenval) [inline test/spec/screenshots.test.js]: grid config ? for multi browser ?

**Comment by mohamedMok** [comment]
@gael-boyenval I'm merging on the tests branch not master yet.

- appli tools access : Waiting on action from @tiagoux 

- documentation (APPLITOOLS_API_KEY ?): Will add a new section for the mozaic doc. I created a task for that (https://design-system-adeo.atlassian.net/browse/MZC-210)

- grid testing: Applitools have a tool for the grid (https://applitools.com/ultrafast-grid)
May worth to have a look. Adding the grid testing today

- dynamic generation of previews list: Focusing on the test on this PR. I created a task to implement it (https://design-system-adeo.atlassian.net/browse/MZC-212)

File: wdio.conf.local.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  maxInstances: 5,
+  sync: false,
+  capabilities: [
+    {
+      browserName: 'chrome',
+      browserVersion: 'latest-1',
+    },
+  ],
+  logLevel: 'trace',
+  baseUrl: 'http://localhost:8000',
```
[the remark this answers is not in the document]
Comment (gael-boyenval) [reply wdio.conf.local.js]: in a local config, the base url may be different right ? 
so maybe we could do something like : 

```
yarn test --baseUrl="http://localhost:8000"
```

you could get the ` --baseUrl` using [`process.argv`](https://nodejs.org/docs/latest/api/process.html#process_process_argv)

**Comment by gael-boyenval** [comment]
> @gael-boyenval I'm merging on the tests branch not master yet.
> 
> * appli tools access : Waiting on action from @tiagoux
> * documentation (APPLITOOLS_API_KEY ?): Will add a new section for the mozaic doc. I created a task for that (https://design-system-adeo.atlassian.net/browse/MZC-210)
> * grid testing: Applitools have a tool for the grid (https://applitools.com/ultrafast-grid)
>   May worth to have a look. Adding the grid testing today
> * dynamic generation of previews list: Focusing on the test on this PR. I created a task to implement it (https://design-system-adeo.atlassian.net/browse/MZC-212)

Why would you not merge it in the master ?

File: wdio.conf.local.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (mohamedMok) [reply wdio.conf.local.js]: https://github.com/adeo/mozaic-design-system/pull/418/commits/f1a506979b28decc161a17d1b57b361cb3af4b31

File: src/docs/GetStarted/Developers/VisualTesting/index.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
```
[the remark this answers is not in the document]
Comment (mohamedMok) [reply src/docs/GetStarted/Developers/VisualTesting/index.mdx]: Make sense! My bad! 
Move it now

**Pull request merged**